### ISSUE TITLE
NullPointerException when getting list of files

### ISSUE THREAD

**Issue by setuper** [dx42/MockFtpServer#20]
```
java.lang.NullPointerException
	at org.mockftpserver.fake.filesystem.AbstractFakeFileSystem.listNames(AbstractFakeFileSystem.java:252)
```
If at the time of calling the listNames method the file was deleted or renamed, then we get NullPointerException
because "getEntry" return null.

**Comment by chrismair** [comment]
Interesting. Thanks.

**Comment by chrismair** [comment]
Fixed in v3.2.0
